applications/StructuralMechanicsApplication/custom_processes/shell_to_solid_shell_process.cpp

```diff
@@ -621,10 +622,12 @@ inline void ShellToSolidShellProcess<TNumNodes>::ComputeNodesMeanNormalModelPart
             aux_coords = this_geometry.PointLocalCoordinates(aux_coords, this_node.Coordinates());
             const array_1d<double, 3>& r_normal = this_geometry.UnitNormal(aux_coords);
             auto& aux_normal = this_node.GetValue(NORMAL);
+
             for (unsigned int index = 0; index < 3; ++index) {
-                #pragma omp atomic
-                aux_normal[index] += r_normal[index];
+                AtomicAdd(aux_normal[index], r_normal[index]);
             }
+            //OR Vector form
+            //AtomicAdd< array_1d<double, 3>, const array_1d<double, 3> >(aux_normal, r_normal);
```

Comment:
This is another way (using vector type AtomicAdd()) , will it work ??

Reply:
It would (I think) but we might change this fct a bit hence I would not use it for now

Reply:
Okay, Fine.